Update availability of Storage Firewall and Service Endpoint for Storage to all regions

SenthuranSivananthan opened the pull request:
Update availability of Storage Firewall and Service Endpoint for Storage to all regions.  Reference:  https://azure.microsoft.com/en-us/updates/public-preview-expansion-vnet-service-endpoints-and-storage-firewalls-and-virtual-networks/

azurecla commented:
Hi __@Juliako__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (Juliako). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

SenthuranSivananthan committed:
Update virtual-network-service-endpoints-overview.md

Update availability of Storage Firewall and Service Endpoint for Storage to all regions.  Reference:  https://azure.microsoft.com/en-us/updates/public-preview-expansion-vnet-service-endpoints-and-storage-firewalls-and-virtual-networks/
